justext/__main__.py

```diff
@@ -6,7 +6,11 @@
 import re
 import os
 import sys
-import cgi
+import six
```

Comment:
Unfortunately `six` is not in the [dependencies](https://github.com/miso-belica/jusText/blob/dev/setup.py#L34-L36), and I think it's not needed. jusText uses a [small compatibility module](https://github.com/miso-belica/jusText/blob/dev/justext/_compat.py) where you can put the code with `escape` (there are more imports like this) and import it here.

There is also [a constant for `PY3`](https://github.com/miso-belica/jusText/blob/dev/justext/_compat.py#L9) you can use here instead of `six.PY2` :)

Reply:
Ok, I'll do that. Thanks you for your review.

By the way, I think `six` must be installed from one of the justText dependencies. I didn't have to explicitly install it in my venv.

Reply:
@miso-belica I amended my latest commit.